### PR TITLE
Update pixhawk3_pro.md

### PR DESCRIPTION
**Pull request by Paul-DROTEK**
Weblinks to Drotek's store added

### CODE REVIEW

File: en/flight_controller/pixhawk3_pro.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ and additional features. The board was designed by [Drotek<sup>&reg;</sup>](http
   * microSD (logging)
   * S.BUS / Spektrum / SUMD / PPM input
   * JST GH user-friendly connectors: same connectors and pinouts as Pixracer
+  
+## Where to buy
+Pixhawk 3 Pro [Pack] : https://store.drotek.com/autopilots/844-pixhawk-3-pro-pack.html
+
+Pixhawk 3 Pro [only] : https://store.drotek.com/autopilots/821-pixhawk-pro-autopilot-8944595120557.html
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk3_pro.md]: 1. Better to do the links hiding bare urls. Ie

```
[Pixhawk 3 Pro (Pack)](https://store.drotek.com/autopilots/844-pixhawk-3-pro-pack.html)

[Pixhawk 3 Pro (board only)](https://store.drotek.com/autopilots/821-pixhawk-pro-autopilot-8944595120557.html)
```

2. Are there any other suppliers - ie Amazon. I appreciate this is your board, but giving options does not hurt.

**Comment by hamishwillee** [comment]
THanks @Paul-DROTEK . Sorry for delay - been away. Merged now.

**Comment by hamishwillee** [comment]
Hi @Paul-DROTEK sorry to necrobump, but I have a [request here](https://discuss.px4.io/t/author-request-reprint-authorization-request/16679) to use the drotek image at the top of the page [here](https://docs.px4.io/master/en/flight_controller/pixhawk3_pro.html).

Technically they can take it under the [documentation license](https://creativecommons.org/licenses/by/4.0/) but would be better for Drotek to confirm they are happy for this to happen :-). Can you confirm?

**Comment by paul-thelu** [comment]
Hi,
Thanks for reaching out.
Of course we agreed to let them use the picture, however would be nice to be able to review the article in which the pic will be used prior its release. :-)

**Pull request merged**